executor/src/witgen/affine_expression.rs

```diff
@@ -19,6 +20,21 @@ pub enum AffineExpression<K, T> {
     ManyVars(Vec<(K, T)>, T),
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Copy)]
+pub enum AlgebraicVariable<'a> {
+    Reference(&'a AlgebraicReference),
```

Comment:
```suggestion
    ColumnReference(&'a AlgebraicReference),
```
or `ColRef`? We could also rename AlgebraicReference.

But we can also do that in a subsequent pr.

Reply:
I renamed it to `Column`! Made sense to me next to `Public`.